Assignment2/Database.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/**
+ * Establish database connection and perform insertion.
+ */
+class Database {
+
+  // Instance variables to store credentials.
+  private $server_name;
+  private $user_name;
+  private $password;
+  private $db_name;
+
+  // Connection object.
+  public $conn;
+
+  // Variables for holding queries.
+  public $q1;
+  public $q2;
+  public $q3;
+
+  /**
+   * Constructor to initialize credentials and establish connection.
+   *
+   * @param string $server
+   *   Server name.
+   * @param string $user
+   *   Username.
+   * @param string $pwd
+   *   User password.
+   * @param string $db
+   *   Database name to connect to.
+   */
+  function __construct(string $server, string $user, string $pwd, string $db) {
+    $this->server_name = $server;
+    $this->user_name = $user;
+    $this->password = $pwd;
+    $this->db_name = $db;
+    try {
+      $this->conn = new mysqli($this->server_name,$this->user_name,$this->password,$this->db_name);
+    }
+    catch (Exception $e) {
+      die('Connection Failed' . $e);
+    }
+  }
+
+  /**
+   * Function to insert form data to database.
+   *
+   * @return bool
+   *   Returns TRUE on data insertion success and false otherwise.
+   */
+  function insertToData() {
+    $this->q1 = "INSERT INTO employee_code_table (
+      employee_code, employee_code_name, employee_domain) values (
+        '{$_POST["employee_code"]}',
```

Comment:
We should first sanitise the data received via user input, validate it and then insert it into the database.
Directly inserting data received via post to db makes it vulnerable to sql injection